sdks/wasm/packager.cs

```diff
@@ -760,6 +763,8 @@ class WasmOptions {
 			emcc_flags += "--llvm-lto 1 ";
 		if (enable_zlib)
 			emcc_flags += "-s USE_ZLIB=1 ";
+		foreach (var pf in preload_files)
+			emcc_flags += "--preload-file " + pf + " ";
```

Comment:
Should the filenames be quoted?

Reply:
I don't think so. The only reason that we might need them is if the pathname to the file contains a space, but in that case there would need to be quotes provided to `packager.exe` anyway or it wouldn't parse the arguments correctly.